sonar-kotlin-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S3353.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "\"var\" should be \"val\" if local variable is never re-assigned",
+  "type": "CODE_SMELL",
+  "status": "ready",
```

Comment:
According to the new [Clean Code Taxonomy](https://xtranet-sonarsource.atlassian.net/wiki/spaces/SONARCLOUD/pages/2880471272/Clean+Code+Taxonomy+for+Rules), we should update the RSPEC to include a code section. For an example, have a look at [this PR](https://github.com/SonarSource/rspec/pull/1645/files#diff-6ca5a191e30de4721e7415a83e1a92f3e543d2f695fa0e174918a26207be244c).

Reply:
:+1: